loops.js

```diff
@@ -18,7 +20,10 @@
 // klaar!
  // ==========================================
 
-
+for (let i = 0; i < 4; i++) {
```

Comment:
Functioneel werkt dit, en geeft het de oplossing. Maar eigen was de bedoeling om een if-statement te gebruiken.

`for (let i = 0; i < 5; i++) {
  if (i === 4) {
    console.log('klaar!')
  } else {
    console.log('loop...');
  }
}`